plugin/server_audit/server_audit.c

```diff
@@ -1448,7 +1458,8 @@ static int log_proxy(const struct connection_info *cn,
                     cn->ip, cn->ip_length,
                     event->thread_id, 0, "PROXY_CONNECT");
   csize+= my_snprintf(message+csize, sizeof(message) - 1 - csize,
-    ",%.*s,`%.*s`@`%.*s`,%d", cn->db_length, cn->db,
+    ",%.*s,%.*s,`%.*s`@`%.*s`,%d", cn->tls_version_length, cn->tls_version,
+                     cn->db_length, cn->db,
```

Comment:
I'm concerned with the compatibility adding tls version early into a CSV output. Machine processing of the list with fail on the later CSV columns like `db` if they assume a structure.

Gut feel is to put at the end. Alternate thoughts welcome.

Reply:
It just feels very weird to have connection information, database name, proxy, host, status and at the end tls_version. But if it's for compatibility, I can make the change